tests/SprykerEcoTest/Zed/Braintree/Business/Order/SaverTest.php

```diff
@@ -92,6 +95,32 @@ public function testSaveOrderPaymentHasAddressData()
         );
     }
 
+    /**
+     * @return void
+     */
+    public function testUpdateOrderPaymentUpdatesPaymentData(): void
+    {
+        // Arrange
+        $saveOrderTransfer = $this->createSaveOrderTransfer();
+        $quoteTransfer = $this->getQuoteTransfer($saveOrderTransfer);
+        $braintreePaymentType = $quoteTransfer
+            ->getPayment()
+            ->getBraintree()
+            ->getPaymentType();
+
+        $idSalesOrder = $saveOrderTransfer->getIdSalesOrder();
+        $this->createPaymentEntity($idSalesOrder);
+
+        $braintreeEntityManager = new Saver(new BraintreeEntityManager());
+
+        // Act
+        $braintreeEntityManager->updateOrderPayment($quoteTransfer, $saveOrderTransfer);
+
+        // Asset
+        $paymentEntity = SpyPaymentBraintreeQuery::create()->findOneByFkSalesOrder($idSalesOrder);
```

Comment:
```suggestion
        $paymentBraintreeEnttiy = SpyPaymentBraintreeQuery::create()->findOneByFkSalesOrder($idSalesOrder);
```

please check variables saming for introduced tests